gradle/verification-metadata.xml

```diff
@@ -664,6 +664,11 @@
             <sha256 value="a42edc9cab792e39fe39bb94f3fca655ed157ff87a8af78e1d6ba5b07c4a00ab" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.google.guava" name="guava" version="32.0.1-jre">
```

Comment:
nit. I'm a bit concerned why we keep also the hashes of the old versions, e.g., why we keep the hashes of the previous 31.1-jre version exactly above? I think we could remove them.

Reply:
If I remove them manually and re-run gradle tooling, those dependencies come back - surely they're transitive for some other libraries